modules/react/button/lib/parts/ButtonLabelIcon.tsx

```diff
@@ -60,6 +60,7 @@ const ButtonLabelIconStyled = styled('span', {
 })<ButtonLabelIconProps>(
   {
     display: 'inline-block',
+    '& > *:first-of-type': {position: 'absolute'},
```

Comment:
My suspicion was confirmed by our visual regression tests. It might be hard to see, but the icons are shifted:

https://www.chromatic.com/test?appId=5d854c26ba934e0020f5e98a&id=61e82d877b910d003adc7b2e

<img width="85" alt="image" src="https://user-images.githubusercontent.com/338257/150177722-d179a3bc-74e1-406b-bceb-b72cfd677198.png">